libfreerdp/core/nla.c

```diff
@@ -54,6 +54,9 @@
 
 #define NLA_AUTH_PKG NEGO_SSP_NAME
 
+static const UINT32 AUTHZ_SUCCESS = 0x00000000;
```

Comment:
nitpicking but in FreeRDP we use enums for these defines

Reply:
ok I never used them in C, I've changed it to an enum